src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -299,7 +302,7 @@ public void deleteRolesOfUserShouldBeSuccess() {
                 .modifyUserToExistingUserForPrdAdmin(
                         HttpStatus.OK, userProfileUpdatedData, intActiveOrgId, invitedUserId);
         assertThat(modifiedUserResponse).isNotNull().hasSize(3);
-        assertThat(((Map)((List)modifiedUserResponse.get("roleDeletionResponse")).get(0)).get("idamStatusCode"))
+        assertThat(((Map) ((List) modifiedUserResponse.get("roleDeletionResponse")).get(0)).get("idamStatusCode"))
                 .isEqualTo("204");
         List<String> rolesToValidate = new ArrayList<>();
```

Comment:
you could use `var` here

Reply:
using `var` here requires further refactoring - as this code is not part of my changes (apart from formatting) I will leave it as it is